miscelanea/003-scc-and-caps/poc-003.yaml

```diff
@@ -0,0 +1,563 @@
+              name: poc-003
+              key: container
+        - name: SYSTEMD_OFFLINE
+          value: "1"
+        - name: SYSTEMD_NSPAWN_API_VFS_WRITABLE
+          value: "network"
+
+      volumeMounts:
+        - name: data
+          mountPath: /data
+        # - name: systemd-run-rpcbind
+        #   mountPath: /run/rpcbind
+        - name: systemd-tmp
+          mountPath: /tmp
+        - name: systemd-sys
+          mountPath: /sys
+          readOnly: true
+        - name: systemd-sys-fs-selinux
+          mountPath: /sys/fs/selinux
+          readOnly: true
+        - name: systemd-sys-firmware
+          mountPath: /sys/firmware
+          readOnly: true
+        - name: systemd-sys-kernel
+          mountPath: /sys/kernel
+          readOnly: true
+        - name: systemd-var-run
+          mountPath: /var/run
+        - name: systemd-var-dirsrv
+          mountPath: /var/run/dirsrv
+
+        # /proc/sys is mounted ro by kubernetes (kubelet?)
+        # proc /proc/sys proc ro,relatime 0 0
+        #
+        # - name: systemd-proc-sys
+        #   mountPath: /proc/sys
+        #   readOnly: true
+        # - name: systemd-proc-sys-net
+        #   mountPath: /proc/sys/net
+
+  volumes:
+    - name: data
+      emptyDir: {}
+    - name: scripts
+      configMap:
+        name: poc-003
+        items:
+          - key: "init-container-uuid.sh"
+            path: "init-container-uuid.sh"
+            mode: 0777
+          - key: "init-uid-gid-base.sh"
+            path: "init-uid-gid-base.sh"
+            mode: 0777
+    # https://github.com/containers/podman/issues/2996
+    #
+    # And from: https://systemd.io/CONTAINER_INTERFACE/
+    # Make sure to pre-mount /proc/, /sys/, and /sys/fs/selinux/ before
+    # invoking systemd, and mount /proc/sys/, /sys/, and /sys/fs/selinux/
+    # read-only in order to prevent the container from altering the host
+    # kernel’s configuration settings. (As a special exception, if your
+    # container has network namespaces enabled, feel free to make
+    # /proc/sys/net/ writable). systemd and various other subsystems (such
+    # as the SELinux userspace) have been modified to behave accordingly
+    # when these file systems are read-only. (It’s OK to mount /sys/ as
+    # tmpfs btw, and only mount a subset of its sub-trees from the real
+    # sysfs to hide /sys/firmware/, /sys/kernel/ and so on. If you do that,
+    # make sure to mark /sys/ read-only, as that condition is what systemd
+    # looks for, and is what is considered to be the API in this context.)
+    - name: systemd-sys
+      hostPath:
+        path: /sys
+        type: DirectoryOrCreate
+    - name: systemd-sys-fs-selinux
+      hostPath:
+        path: /sys/fs/selinux
+        type: Directory
+    - name: systemd-sys-firmware
+      hostPath:
+        path: /sys/firmware
+        type: Directory
+    - name: systemd-sys-kernel
+      hostPath:
+        path: /sys/kernel
+        type: Directory
```

Comment:
This is one block to investigate for removing `allowHostDirVolumePlugin: true` from the SCC object.